src/qt/receiverequestdialog.cpp

```diff
@@ -45,7 +45,7 @@ QImage QRImageWidget::exportImage()
 {
     if(!pixmap())
         return QImage();
-    return pixmap()->toImage().scaled(EXPORT_IMAGE_SIZE, EXPORT_IMAGE_SIZE);
+    return pixmap()->toImage().scaled(EXPORT_IMAGE_SIZE, EXPORT_IMAGE_SIZE, Qt::KeepAspectRatio, Qt::SmoothTransformation);
```

Comment:
QRCodes should not use anti-aliasing (QT::SmoothTransformation). I guess this should be removed.